src/planwise/model/import_job.clj

```diff
@@ -390,6 +390,10 @@
                          :type-field type-field)]
     (import-job job-value)))
 
+(defn restore-job
+  [job]
+  (import-job (:state job) (:value job)))
```

Comment:
The job is only serialised once a task is completed/failed, so if there are any pending tasks, these should be re-created when the job is restored and next-task is called. I can add a test for that to double-check.